src/components/Form/Form.tsx

```diff
@@ -0,0 +1,151 @@
+import "./style.scss";
+import { useFormik } from "formik";
+import { Modal } from "../../interfaces";
+import { films } from "../../films";
+import { CONSTANTS } from "../../constants";
+import Select from "react-select";
+import DeleteForm from "./DeleteForm";
+
+const selectStyle: any = {
+  menu: () => {
+    return {
+      backgroundColor: "#232323;",
+      position: "absolute",
+      margin: "20px",
+      marginTop: "0",
+      display: "block",
+      width: "330px",
+      top: "48px",
+      borderRadius: "3px",
+    };
+  },
+  option: () => {
+    return {
+      ":hover": {
+        backgroundColor: "#f65261;",
+      },
+    };
+  },
+  control: () => {
+    return {
+      backgroundColor: "#555",
+      margin: "20px",
+      border: "none",
+      height: "26px",
+      borderRadius: "3px",
+      color: "white",
+      display: "flex",
+    };
+  },
+};
+
+interface FormProps {
+  closeModal(): void;
+  modalState: Modal;
+}
+
+const Form: React.FC<FormProps> = ({ closeModal, modalState }) => {
+  const movie: any =
+    modalState.type === CONSTANTS.FORM_TYPE.ADD
```

Comment:
not the best way, CONSTANTS.FORM_TYPE.ADD,
better to have object or enum, without wrapper CONSTANTS
```
FormType = {
ADD:
..
```